index.html

```diff
@@ -35,7 +35,7 @@
                         <img src="assets/bluesky-logo-dark.svg" alt='Bluesky'/>
                         <p class="standout">
                             <!-- {Presenter Name} and <br /> -->
-                            The Bluesky Project Contributors<br />(NSLS-II, APS, LCLS-II, ALS)
+                            The Bluesky Project Contributors
```

Comment:
As noted in the commit message, the concern here is leaving out folks from other facilities and labs who have made contributions to the project.